fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1312,6 +1312,7 @@ void SimulationConfig::setStorageEngine(const TestConfig& testConfig) {
 	// Using [0, 4) to disable the RocksDB storage engine.
 	// TODO: Figure out what is broken with the RocksDB engine in simulation.
 	int storage_engine_type = deterministicRandom()->randomInt(0, 4);
+	// int storage_engine_type = 4; // Focus test to RocksDB engine
```

Comment:
Is this intended?

Reply:
yes. making it obvious how to focus on rocksdb sim test